packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -206,6 +209,7 @@ const withChunking = (config) => {
 
 const ruleForBabelLoader = (babelPlugins) => {
     return {
+        id: 'javascript',
```

Comment:
`javascript` is the target language for this rule but I feel more natural to call this the `babel-loader`, I feel like `babel-loader` kinda implies it is js/ts and it is also a little bit more descriptive?